dynatrace/0.20.0/artifacthub-pkg.yml

```diff
@@ -0,0 +1,156 @@
+  * Whether to pull SLI/SLO information from a Dynatrace dashboard or use the stored\
+  \ `sli.yaml` and `slo.yaml` in the Keptn Configuration Repository\n\nHere is an\
+  \ example `dynatrace.conf.yaml`:\n\n```yaml\nspec_version: '0.1.0'\ndtCreds: dynatrace-preprod\n\
+  dashboard: query\n```\n\nTo upload this to your Keptn project you can for instance\
+  \ use the Keptn CLI:\n\n```console\nkeptn add-resource --project=yourproject --stage=yourstage\
+  \ --resource=./dynatrace.conf.yaml --resourceUri=dynatrace/dynatrace.conf.yaml\n\
+  ```\n\n**`dtCreds`**\n\n`dtCreds` allows you to specify the name of the k8s secret\
+  \ in your Keptn namespace that holds the required credentials to connect to the\
+  \ Dynatrace Tenant. This extends the default behavior as explained in the beginning\
+  \ by having the *dynatrace-service* first look at the secret defined in dtCreds.\
+  \ If `dtCreds` is not specified or if there is no `dynatrace.conf.yaml` at all then\
+  \ it just does the default behavior.\n\nIn the example above where `dtCreds` was\
+  \ specified with the value *dynatrace-preprod* the *dynatrace-service* would be\
+  \ looking for the first matching secret in the following order: *dynatrace-preprod*,\
+  \ *dynatrace-credentials-YOUR-KEPTN-PROJECT*, *dynatrace-credentials*, *dynatrace*\n\
+  If none of these secrets is configured in your k8s Keptn namespace the *dynatrace-service*\
+  \ will respond with an error indicating that no Dynatrace credentials could be found!\n\
+  \nFor completeness, here is an example of how to create a secret that matches the\
+  \ `dynatrace.conf.yaml`:\n\n```console\nkubectl create secret generic dynatrace-preprod\
+  \ -n \"keptn\" --from-literal=\"DT_TENANT=$DT_TENANT\" --from-literal=\"DT_API_TOKEN=$DT_API_TOKEN\"\
+  \n```\n\n`dtCreds` was requested by many users as it gives you the option to specify\
+  \ credentials for your different Dynatrace Tenants, e.g. my-dynatrace-preprod, my-dynatrace-prod,\
+  \ my-dynatrace-dev. And then you can configure on project, stage or even service\
+  \ level which Dynatrace Tenant to be used. This gives you all flexiblity to manage\
+  \ multiple environments within a single project but separate it out by e.g. stages.\n\
+  \n## Up- or Downgrading\n\nAdapt and use the following command in case you want\
+  \ to up- or downgrade your installed version (specified by the `$VERSION` placeholder):\n\
+  \n```console\nhelm upgrade dynatrace-service -n keptn https://github.com/keptn-contrib/dynatrace-service/releases/download/$VERSION/dynatrace-service-$VERSION.tgz\n\
+  ```\n\n## Uninstall\n\nTo delete a deployed *dynatrace-service*, use the Helm CLI\
+  \ to uninstall the installed release of the service:\n\n```console\nhelm delete\
+  \ -n keptn dynatrace-service\n```\n"
+recommendations:
+- url: https://artifacthub.io/packages/helm/keptn/keptn
+annotations:
+  keptn/kind: observability
+  keptn/org: contrib
+  keptn/version: 0.8.4-0.9.2
```

Comment:
This needs to be updated urgently!
@arthurpitman @j-poecher what's the desired compatibility you want to be presented to the user?

Reply:
latest release is compatible with Keptn `0.11.x`